engines/dfc_provider/app/services/enterprise_importer.rb

```diff
@@ -40,5 +44,41 @@ def apply(enterprise)
       state: state,
       country: state.country,
     )
+    enterprise.email_address = @dfc_enterprise.emails.first
+    enterprise.description = @dfc_enterprise.description
+    enterprise.phone = @dfc_enterprise.phoneNumbers.first&.phoneNumber
+    enterprise.website = @dfc_enterprise.websites.first
+    apply_social_media(enterprise)
+    apply_logo(enterprise)
+  end
+
+  def apply_social_media(enterprise)
+    attributes = {}
+    @dfc_enterprise.socialMedias.each do |media|
+      attributes[media.name.downcase] = media.url
+    end
+    attributes["twitter"] = attributes.delete("x") if attributes.key?("x")
+    enterprise_attributes = attributes.slice(SocialMediaBuilder::NAMES)
+    enterprise.assign_attributes(enterprise_attributes)
+  end
+
+  def apply_logo(enterprise)
+    link = @dfc_enterprise.logo
+    logo = enterprise.logo
+
+    return if link.blank?
+    return if logo.blob && (logo.blob.custom_metadata&.fetch("origin", nil) == link)
```

Comment:
"Don't re-fetch the logo if we have already downloaded it from that URL". I like it!